src/js/tracks/text-track.js

```diff
@@ -19,7 +19,25 @@ import XHR from 'xhr';
  * @param {String} srcContent webVTT file contents
  * @param {Track} track track to addcues to
  */
-const parseCues = function(srcContent, track) {
+const parseCues = function(srcContent, track, retryNum) {
+  // Make sure that vttjs has loaded, otherwise, wait till it finished loading
+  // NOTE: this is only used for the alt/video.novtt.js build
+  if (typeof window.WebVTT !== 'function') {
+    let retry;
+
+    if (retryNum === 0) {
+      return log.error(`vttjs did not load, stopping trying to process ${track.src}`);
+    } else if (typeof retryNum !== 'number') {
+      retry = 3;
+    } else {
+      retry = retryNum - 1;
+    }
+
+    return window.setTimeout(function() {
+      parseCues(srcContent, track, retry);
+    }, 100);
+  }
```

Comment:
@chemoish looks like it's not as urgent as other things, so, I'll spend time making it wait for vttjs to load instead of retrying :)